client/src/main/scala/com.gu.contentapi.client/model/Queries.scala

```diff
@@ -8,9 +8,16 @@ sealed trait ContentApiQuery {
 
   def pathSegment: String
 
-  override def toString = {
-    s"${getClass.getSimpleName}(/$pathSegment${QueryStringParams(parameters)})"
+  override def toString = apply("")
```

Comment:
This needs to be `s"${getClass.getSimpleName}${apply("")}"` if we want to include the class name.

Reply:
I can see now in the tests that removing the class name was a conscious choice.